CHANGELOG.md

```diff
@@ -1,3 +1,68 @@
+# 93.5.0
+
+This minor release introduces new APIs for Slider. The track height and the
+visibility of track tick marks can be customized.
+
+## New features
+
+`MDCSlider` allows customization of the track height.
```

Comment:
```
`MDCSlider` now allows customization
```

To emphasize that this is new functionality, not existing functionality (as presently written).

Reply:
Thanks, Jeff. We spoke offline and agreed that correcting this wasn't worth the trade-off in time for presubmits given the urgent deadline for this Hotfix.